### DISCUSSION TITLE
copyright is not at the bottom?

### DISCUSSION THREAD

**Discussion by AkagiYui** [facebook/docusaurus#7756] [Q&A] [answered]
The footer or copyright is not at the bottom of window.
I found this problem when I update to `2.0.0-beta.22`

old: https://blog-dvuvgqvva-akagiyui.vercel.app/docs/public_api/
![image](https://user-images.githubusercontent.com/39934721/178138755-0f14709a-7096-4314-a33e-fdf9144be172.png)

new: https://blog-fahs12p45-akagiyui.vercel.app/docs/public_api/
![image](https://user-images.githubusercontent.com/39934721/178138773-c32cf0a3-0c8f-4739-a1d8-2f113437af78.png)

But I cannot this problem from https://docusaurus.io/docs/category/getting-started.
I did something wrong?

**Comment by Josh-Cena** [comment, marked as answer]
Should be fixed by https://github.com/facebook/docusaurus/pull/7752 already. [Try canary?](https://docusaurus.io/community/canary)

**Reply by AkagiYui** [reply]
Thanks, it works.
